fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/cursors/SizeStatisticsResultsTest.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * SizeStatisticsResultsTest.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.provider.foundationdb.cursors;
+
+import com.apple.foundationdb.KeyValue;
+import com.apple.foundationdb.record.RecordCoreArgumentException;
+import com.apple.foundationdb.record.RecordCursorProto;
+import com.google.common.base.Strings;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+/**
+ * Unit tests for {@link SizeStatisticsResults}.
+ */
+public class SizeStatisticsResultsTest {
+
+    @Test
+    public void emptyStatistics() {
+        SizeStatisticsResults stats = new SizeStatisticsResults();
+        
+        Assertions.assertThat(stats.getKeyCount()).isZero();
```

Comment:
The codebase is somewhat inconsistent about this, but we should be able to statically import `Assertions.assertThat` to make this a little bit less verbose